### PR TITLE
add more READMEs

### PR DESCRIPTION
**Pull request by hopkiw**
Add README files for the container images and package build workflows

### CODE REVIEW

File: container_images/README.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+# Docker container images
+
+The source for Docker container images used in our CI/CD infrastructure. Each
+folder represents one image. These are built by kaniko jobs in our Concourse
```
Comment (Linskeyd) [inline container_images/README.md:4]: Nit: would be useful to link to what kaniko is for folks that don't know.

File: container_images/README.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+[oss-test-infra]: https://github.com/GoogleCloudPlatform/oss-test-infra/tree/master/prow/prowjobs/GoogleCloudPlatform/gcp-guest/
+
+## Images
+
+A brief description of each container image is listed below.
+
+### build-essential
+
+This is a clone of a Debian container with the 'build-essential' package
+preinstalled. Usage: ??
```
Comment (Linskeyd) [inline container_images/README.md:23]: For this and other 'Usage: ??' locations -> please fill this in or remove for the merge in.

File: container_images/README.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+A fork of [concourse/registry-image-resource] that supports GCE authentication.
+Used in Concourse pipelines to track private images such as those built in this
+repository.
+
+[concourse/registry-image-resource]: https://github.com/concourse/registry-image-resource
+
+### selinux-tools
+
+SELinux tools for building SELinux modules. Currently broken, see issue: blah
```
Comment (Linskeyd) [inline container_images/README.md:105]: blah

File: packagebuild/README.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
+# Package build workflows
+
+This directory contains the 'package builder', which is a set of [Daisy]
```
Comment (Linskeyd) [inline packagebuild/README.md:3]: Missing a Daisy link

**Review by Linskeyd — CHANGES_REQUESTED** [review]

File: container_images/README.md (hunk 2 above)
Comment (hopkiw) [reply container_images/README.md:23]: no we want to keep that bc this is documenting state. some of these have no known usage

**Pull request merged**